File: generated-sources/kotlin-spring/mojang-authentication/src/main/kotlin/com/github/asyncmc/mojang/authentication/kotlin/spring/model/GameProfileProperty.kt
package com.github.asyncmc.mojang.authentication.kotlin.spring.model

import java.util.Objects
import com.fasterxml.jackson.annotation.JsonProperty
import javax.validation.Valid
import javax.validation.constraints.*

/**
 * 
 * @param name 
 * @param value 
 */
data class GameProfileProperty (

        @JsonProperty("name") val name: String? = null,

        @JsonProperty("value") val value: String? = null
) {

}

File: generated-sources/scala-gatling/mojang-authentication/src/gatling/scala/com/github/asyncmc/mojang/authentication/scala/gatling/model/GameProfileProperty.scala

package com.github.asyncmc.mojang.authentication.scala.gatling.model


case class GameProfileProperty (
    _name: Option[String],
    _value: Option[String]
)
object GameProfileProperty {
    def toStringBody(var_name: Object, var_value: Object) =
        s"""
        | {
        | "name":$var_name,"value":$var_value
        | }
        """.stripMargin
}
